dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/MemoryInformation.java

```diff
@@ -896,7 +895,7 @@ private synchronized long getTotalMemoryJDK() throws MemoryInformationException
         try {
             ensureInitializedMBean();
 
-            if (JavaInfo.vendor() == Vendor.IBM) {
+            if (JavaInfo.isAvailable("com.ibm.security.auth.module.Krb5LoginModule")) {
```

Comment:
Is checking for Krb5LoginModule the best we can do?

Reply:
I agree, this seems unconnected to the method which is being called.

Reply:
I'm not very familiar with MBeans. Would calling `MBeanServer.getAttributes` to request both attributes and seeing which one exists from the result be better here?

Reply:
I did consider that option too. This can definitely be improved but under a separate PR.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/18052